proto/minder/v1/minder.proto

```diff
@@ -2472,9 +2499,18 @@ message Role {
 
 message RoleAssignment {
     // role is the role that is assigned.
-    string role = 1;
+    string role = 1 [
+        (buf.validate.field).string = {
+            pattern: "^[a-z]+(_[a-z]+)*$",
+            min_len: 1,
+            max_len: 200,
+        }
+    ];
     // subject is the subject to which the role is assigned.
-    string subject = 2;
+    string subject = 2 [
+        (buf.validate.field).string = {uuid: true},
```

Comment:
I'm not sure these are always UUIDs, but they may happen to be UUIDs for now.  I think it's okay to leave this as-is at the moment.